application-admintools-ui/src/main/resources/AdminTools/Code/Translations.xml

```diff
@@ -36,23 +36,40 @@
   <syntaxId>plain/1.0</syntaxId>
   <hidden>true</hidden>
   <content>##Admin section
+AdminTools.Code.ConfigurationClass_excludedLines=Excluded lines
+AdminTools.Code.ConfigurationClass_excludedLines.hint=Lines containing sensitive info in xwiki.cfg and xwiki.properties files
 AdminTools.Code.ConfigurationClass_serverLocation=Server location
 AdminTools.Code.ConfigurationClass_serverLocation.hint=Path to the server location
 admin.admintools.description=Configure Admin Tools Application
+adminTools.dashboard.modal.button.close=Close
```

Comment:
You could match them into sections, by leaving a new line between them https://github.com/xwikisas/application-collabora/blob/main/application-collabora-ui/src/main/resources/Collabora/Code/Translations.xml#L51